src/compiler/crystal/tools/playground/server.cr

```diff
@@ -1,23 +1,29 @@
 require "http/server"
-require "logger"
+require "log"
 require "ecr/macros"
 require "compiler/crystal/tools/formatter"
 require "compiler/crystal/tools/doc/markdown"
 
 module Crystal::Playground
+  Log = begin
+    builder = ::Log::Builder.new
+    builder.bind("crystal.playground", :warning, ::Log::StdioBackend.new)
+    builder.for("crystal.playground")
+  end
```

Comment:
Why isn't this just `Log = ::Log.for("crystal.playground", :warning)`?
This doesn't look like the pattern we want the users to embrace

Reply:
The playground is the only compiler module that logs something. `crystal play` logs warnings only and `crystal play -v` changes the log level to debug.

The there is a need for a custom logger initialization to keep that.

This is the change to keep the usage of the logger isolated in the playground.

The alternative would be to Use `Log = ::Log.for("crystal.playground")`. Define that the compiler configs the logger with some policy. Allow the playground cli to change the log level of `crystal.playground` logger.

We can do that if prefered.

Reply:
The commit https://github.com/crystal-lang/crystal/pull/8847/commits/56d0e6aa83045adc54499dc9cb79c19b382db359 enables the logger on the whole compiler only for `crystal.*` sources and warning level by default. The `crystal play -v` will change the playground log level only. The CRYSTAL_LOG_LEVEL/CRYSTAL_LOG_SOURCES can be used to tweak the logger configuration further.

Reply:
Thanks, I like this solution!